cmd/util/ledger/migrations/change_contract_code_migration_test.go

```diff
@@ -0,0 +1,200 @@
+package migrations_test
+
+import (
+	"context"
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/flow-go/cmd/util/ledger/migrations"
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/common/convert"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestChangeContractCodeMigration(t *testing.T) {
+
```

Comment:
Here and for subtests:

```suggestion
	t.Parallel()

```